src/test/suite/fixtures/recorded/updateSelections/bringFine.yml

```diff
@@ -0,0 +1,35 @@
+spokenForm: bring fine
```

Comment:
The tests in this directory are all selection based, because we now control cursor in response to a bring.  For example, in this one we check that the selection is moved to the right